### PR TITLE
[#1394] marked more elements to be removed in 2.22

### PR DESCRIPTION
**Pull request by cdietrich**
[#1394] marked more elements to be removed in 2.22
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

@JanKoehnlein please check if i missed to mark something

### CODE REVIEW

File: org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/coloring/IColoringService.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
  * 
  * @author akos.kitta - Initial contribution and API
  * @deprecated use the {@link ISemanticHighlightingCalculator} and the {@link ISemanticHighlightingStyleToTokenMapper} instead.
+ * 			   This method is scheduled to be removed with 2.22.
```
Comment (szarnekow) [inline org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/coloring/IColoringService.java]: `This method` should be `This interface`

File: org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/codeActions/ICodeActionService.xtend [hunk 2]
```diff
@@ -22,8 +22,9 @@ import org.eclipse.xtext.util.CancelIndicator
 /**
  * @author Sven Efftinge - Initial contribution and API
  * @deprecated use {@link ICodeActionService2} instead
+ *             This method is scheduled to be removed with 2.22.
  */
-@Deprecated
+@Deprecated//(forRemoval=true)
 interface ICodeActionService {
```
Comment (szarnekow) [inline org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/codeActions/ICodeActionService.xtend]: `This interface`

File: org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/ChangeConverter.xtend [hunk 3]
```diff
@@ -28,8 +28,9 @@ import org.eclipse.xtext.util.IAcceptor
  * @author koehnlein - Initial contribution and API
  * @since 2.13
  * @deprecated use {@link ChangeConverter2} instead
+ *             This method is scheduled to be removed with 2.22.
```
Comment (szarnekow) [inline org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/ChangeConverter.xtend]: `This class`

File: org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/IRenameService.java [hunk 4]
```diff
@@ -17,8 +17,9 @@
  * @author koehnlein - Initial contribution and API
  * @since 2.13
  * @deprecated use {@link IRenameService2} instead.
+ *             This method is scheduled to be removed with 2.22.
```
Comment (szarnekow) [inline org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/IRenameService.java]: `This interface`

File: org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/RenameService.xtend [hunk 5]
```diff
@@ -29,8 +29,9 @@ import org.eclipse.xtext.resource.impl.ResourceDescriptionsProvider
  * @author koehnlein - Initial contribution and API
  * @since 2.13
  * @deprecated use {@link RenameService2} instead.
+ *             This method is scheduled to be removed with 2.22.
  */
-@Deprecated
+@Deprecated//(forRemoval=true)
```
Comment (szarnekow) [inline org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/RenameService.xtend]: `This class`

**Comment by cdietrich** [comment]
@szarnekow updated with replacements method->class/if

**Comment by JanKoehnlein** [comment]
Sorry to be late to the party. LGTM

**Pull request merged**